BUILDING.md

```diff
@@ -117,8 +119,8 @@ To run the tests:
 
 To test if Node.js was built correctly:
 
-```
-$ node -e "console.log('Hello from Node.js ' + process.version)"
+```text
+> Release\node -e "console.log('Hello from Node.js ' + process.version)"
```

Comment:
might be nice to update this, either `console.log(`Hello from Node.js ${process.version}`)` or `console.log('Hello from Node.js', process.version)`